JACK backend uses sched_yield() on windows

Compiling libremidi on windows results in error: 
libremidi-5.4.3\include\libremidi/backends/jack/helpers.hpp(282): error C3861: 'sched_yield': identifier not found

--- a/include/libremidi/backends/jack/helpers.hpp
+++ b/include/libremidi/backends/jack/helpers.hpp
@@ -8,6 +8,7 @@
 #include <cstdint>
 #include <cstring>
 #include <memory>
+#include <thread>
 
 NAMESPACE_LIBREMIDI
 {
@@ -279,7 +280,7 @@ public:
       return std::errc::no_buffer_space;
 
     while (jack_ringbuffer_write_space(ringbuffer) < sz + size_sz)
-      sched_yield();
+      std::this_thread::yield();
 
     jack_ringbuffer_write(ringbuffer, reinterpret_cast<char*>(&sz), size_sz);
     jack_ringbuffer_write(ringbuffer, reinterpret_cast<const char*>(data), sz);

